serde-generate/src/java.rs

```diff
@@ -485,28 +488,29 @@ fn output_struct_or_variant_container(
     nested_class: bool,
     package_prefix: &str,
 ) -> Result<()> {
-    let tab = " ".repeat(indentation);
+    let mut out = IndentedWriter::new(out, Indent::Space(4));
+    out.set_level(indentation / 4);
     // Beginning of class
     if let Some(base) = variant_base {
+        writeln!(out)?;
         writeln!(
             out,
-            "\n{}public static final class {} extends {} {{",
```

Comment:
I don't think you can remove the empty new line yet.

Reply:
I added a new `writeln!` call jus before this, in order to make it clearer